microsoft-edge/toc.yml

```diff
@@ -1029,89 +1046,181 @@
+#           - name: User Ratings  # how User Ratings work, eg how region-wide users can see the review & ratings
+#             href:
 
-            - name: Policies for Microsoft Edge Add-ons
-              items:
-                - name: Content Security Policy (CSP)
-                  href: extensions-chromium/store-policies/csp.md
+#           - name: Review Process  # the review process for an extension that's submitted to add-ons site
+#             href:
 
-                - name: App Developer Agreement Addendum for Microsoft Edge program users
-                  href: extensions-chromium/store-policies/ada-addendum.md
+#           - name: Retire your Extension  # how to uninstall or retire an extension
+#             href:
 
-                - name: Developer policies for the Microsoft Edge Add-ons store
-                  href: extensions-chromium/store-policies/developer-policies.md
+# -------------------------------------
+        - name: Best practices
+          items:
+#           - name: Checklist for publishing an extension
+#             href:
 
-            - name: Microsoft Edge Add-ons API
-              items:
-                - name: Using the Microsoft Edge Add-ons REST API
-                  href: extensions-chromium/publish/api/using-addons-api.md
+            - name: Best practices for extensions
+              href: extensions-chromium/developer-guide/best-practices.md
 
-                - name: REST API Reference for Microsoft Edge Add-ons
-                  href: extensions-chromium/publish/api/addons-api-reference.md
+#           - name: Promote an extension
+#             href:
 
+#           - name: The Featured badge
+#             href:
+
+#           - name: Policy checklist
+#             href:
+
+#           - name: How to build non-impacting extensions  # things that can impact speed of browser and extension
+#             href:
+
+#           - name: Things to avoid, that can get an extension rejected
+#             href:
+
+# -------------------------------------
         - name: Samples
-          href: extensions-chromium/samples.md
-          displayName: Samples for Microsoft Edge extensions # full title
+          items:
+            - name: Samples for Microsoft Edge extensions
+              href: extensions-chromium/samples.md
+
+#           - name: Sample extension using a content script
+#             href:
+
+#           - name: Sample extension using a service worker
+#             href:
+
+#           - name: Sample extension using the Sidepanel API
+#             href:
+
+# -------------------------------------
+        - name: Support and feedback
+          items:
+            - name: Contact Microsoft Edge extensions support
+              href: extensions-chromium/publish/contact-extensions-team.md
+              displayName: feedback, log issues, enter issues, report issues, reporting issues
+
+#           if h1's, instead of h2's in the above h1:
```

Comment:
should the following 4 nodes be h1's (ie, distinct articles / .md files), or remain h2 sections in the above article?
Rendered: 
https://review.learn.microsoft.com/en-us/microsoft-edge/extensions-chromium/publish/contact-extensions-team?branch=pr-en-us-2899

@sivMSFT @Bhuvana-Priya

Reply:
I might push a commit to implement a consistent, specific design - probably the best interpretation of Siv's July 3 draft outline (consistent w/ design of those other feature areas; consistent w/ the entire Edge Dev docset's TOC) is that there's a top-level "Contact" article (h1 title) (not a Support TOC bucket) which contains 4 h2 headings/sections: 
* Microsoft Edge extensions
   * Contact Microsoft Edge extensions support (h1 article title)
      * Request help  (h2 heading/section)
      * Report copyright  (h2 heading/section)
      * Log tickets  (h2 heading/section)
      * Connect community (h2 heading/section)

Reply:
existing h2 sections in live Contact article approximately matches my proposed outline design:
![image](https://github.com/user-attachments/assets/91099283-e628-468b-8d3d-0b5581c5d7ac)

Reply:
promoted Contact article, which replaced Support TOC bucket.  Moved subheadings of draft outline from `toc.yml` file to `contact-extensions-team.md`, as a hidden comment to become a PR Comment.